src/app/CommandHandlerImpl.cpp

```diff
@@ -58,14 +58,17 @@ CommandHandlerImpl::~CommandHandlerImpl()
 
 CHIP_ERROR CommandHandlerImpl::AllocateBuffer()
 {
+    System::PacketBufferHandle commandPacket = nullptr;
```

Comment:
This should be declared where first used, not up here.

Reply:
Done.